source/src/third/cuckoo/CuckooFilter.cpp

```diff
@@ -294,9 +219,9 @@ bool CuckooFilter::remove(ExtendedFingerprint efp) {
 	  //    on fail try B,
 	  //    if either succes, fix victim.
 
-	  if(victim != 0) {
+	  if(victim.fingerprint != 0) {
 		  if(add(victim)) {
-			  victim = 0;
+			  victim = ExtendedFingerprint{};
```

Comment:
what does this do?

Reply:
construct default ExtendedFingerprint value and assign it to victim.

Reply:
Isn't that done with `ExtendedFingerprint();` ?

Reply:
Yup can be used too, but there's a raging debate on best practices in c++ concerning initialization syntax. There are some cases where the brace init seems to be favorable, for example because it forbids narrowing conversions between arithmetic types. E.g.:
```
char x = 256; // legal
char x{256}; // illegal
```
To me it looks more clear to be a constructor call rather than a function call.

Reply:
Most languages use ClassName() for instantiation though, right? So it was just a bit confusing to me.

Reply:
This also falls under https://en.cppreference.com/w/cpp/language/aggregate_initialization .